CONTRIBUTING.md

```diff
@@ -9,7 +9,7 @@ StackOverflow is a much better place to ask questions since:
 * questions and answers stay available for public viewing so your question / answer might help someone else
 * SO voting system assures that the best answers are prominently visible.
 
-To save your and our time we will be systematically closing all the issues that are request for general support and redirecting people to StackOverflow. 
+To save your and our time we will be systematically closing all the issues that are request for general support and redirecting people to StackOverflow.
```

Comment:
I didn't change the text at all - I think my editor just chopped off useless whitespace.

I think the former point, although a little verbose, is a little less harsh than the proposed amendment.

I'll change the `request` part though.